grammar.js

```diff
@@ -31,6 +31,8 @@ module.exports = grammar(C, {
     [$._declaration_modifiers, $.operator_cast_declaration, $.operator_cast_definition, $.constructor_or_destructor_definition],
     [$._declaration_modifiers, $.attributed_statement, $.operator_cast_declaration, $.operator_cast_definition, $.constructor_or_destructor_definition],
     [$.attributed_statement, $.operator_cast_declaration, $.operator_cast_definition, $.constructor_or_destructor_definition],
+    [$._declaration_modifiers, $.type_descriptor],
+    [$._declaration_specifiers, $.type_descriptor],
```

Comment:
need to check whether this is necessary. Conflict is generated by using `parameter_list` in requires clause

Reply:
Were you able to figure out if you could change the dynamic precedence to not need these?

Reply:
I could get away when I give either type_descriptor/ repeat($.type_qualifier) or _declaration_modifiers/_declaration_specifiers a higher precedence. However, this causes the one Attributes test to fail.

Reply:
with dynamic precedence you mean `prec(X, ...)` ?

Reply:
I meant `prec.dynamic(X, ...)` but I looked at this a bit more and I think it's appropriate to leave this here in the conflicts.